src/dotnet/ToolPackage/ToolPackageObtainer.cs

```diff
@@ -172,7 +174,11 @@ private static LockFileItem FindAssetInLockFile(LockFile lockFile, string toolCo
                         new XElement("RestorePackagesPath", individualToolVersion.Value),
                         new XElement("RestoreProjectStyle", "DotnetToolReference"),
                         new XElement("RestoreRootConfigDirectory", Directory.GetCurrentDirectory()),
-                        new XElement("DisableImplicitFrameworkReferences", "true")
+                        new XElement("DisableImplicitFrameworkReferences", "true"),
```

Comment:
Can you add comments explaining why these properties are set to these values?

Reply:
Fixed, added comment